packages/test-utils/jest-spyOn.ts

```diff
@@ -1,4 +1,4 @@
-export const jestMock = (object: any, method: string, implementation: () => any, accessType?: 'get' | 'set') => {
+export const jestSpyOn = (object: any, method: string, implementation: () => any, accessType?: 'get' | 'set') => {
   let mockSpy = null
   if(!accessType){
```

Comment:
This condition judgement seems redundant. `jest.spyOn` already has take `accessType: undefined` into condiseration

Reply:
The declear of this function is below:
![image](https://user-images.githubusercontent.com/33176053/92068756-57ce1b00-edda-11ea-89c6-4572a2c9e85d.png)

I think it is more clear though the implement of `spyOn` already check the `accessType`

Reply:
> The declear of this function is below:
> ![image](https://user-images.githubusercontent.com/33176053/92068756-57ce1b00-edda-11ea-89c6-4572a2c9e85d.png)
> 
> I think it is more clear though the implement of `spyOn` already check the `accessType`

You can do it with ts declaration too but not in function implemention. BTW, I still do NOT think it's necessary to extract `jestSpyOn`

Reply:
Actually Jerry Wuuuuu asked me to extract it, I'll take a look again and delete it if it is not necessary.

Reply:
emmm , i dont think it's necessary too.

Reply:
OK, I will delete this file in this PR later.